launcher/electron/app/styles/dialog.css

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
These dialogs are used by Electron to control both Beaker's frontend and backend. As of now, the only instance of this dialog is to switch backend (switch to a backend at a different URL) or to kill the current backend and start a new one. Then you can't reuse anything, since this dialog has to be independent of Beaker.